fastlane/lib/fastlane/actions/upload_symbols_to_crashlytics.rb

```diff
@@ -10,8 +10,8 @@ def self.run(params)
         find_gsp_path(params)
         find_api_token(params)
 
-        if !params[:api_token] && !params[:gsp_path]
-          UI.user_error!('Either Fabric API key or path to Firebase Crashlytics GoogleService-Info.plist must be given.')
+        if !params[:api_token] && !params[:gsp_path] && !params[:app_id]
```

Comment:
Nitpick/OCD:
```suggestion
        if !params[:app_id] && !params[:gsp_path] && !params[:api_token]
```
Same order as in error message ;)

Reply:
Makes perfect sense to me 🙏